Dockerfile

```diff
@@ -12,8 +12,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-ARG BASE_IMAGE=nvcr.io/nvidia/tritonserver:23.08-py3
```

Comment:
I don't know why these changes are showing up after I rebased, but these and the docs changes are safe to ignore.

Reply:
Maybe you can try this to remove the unwanted commit from PR: https://stackoverflow.com/a/51400593

Reply:
I'm not too concerned. They match what is on the `add-llm-mode`  branch and will go away when this subbranch is pushed. Just wanted to call it out so you wouldn't review it.